packages/combobox/src/Combobox.ts

```diff
@@ -402,6 +440,7 @@ export class Combobox extends Textfield {
                     : ''}"
                 ?disabled=${this.disabled}
                 ?focused=${this.focused}
+                ?quiet=${this.quiet}
```

Comment:
not really related to this PR but a very straight-forward fix for the `quiet` state